.devcontainer/Dockerfile

```diff
@@ -89,6 +89,7 @@ RUN apt-get update && \
 RUN pip install ipython jupyter jupyterlab
 RUN pip install python-dotenv simplejson
 
+RUN pip install cython
```

Comment:
For some reason I was getting failures in rebuilding the dev container, that said that scipy couldn't compile because cython wasn't available. This seems sort of contrary to the idea of a dependency manager, but adding an explicit installation of cython seemed to fix it, so... ??

Reply:
Looks good